pkg/querier/distributor_queryable_test.go

```diff
@@ -261,6 +262,93 @@ func TestIngesterStreamingMixedResults(t *testing.T) {
 	require.NoError(t, seriesSet.Err())
 }
 
+func genTestHistogram(timestamp int64, value int) mimirpb.Histogram {
+	return mimirpb.FromHistogramToHistogramProto(timestamp, test.GenerateTestHistogram(value))
+}
+
+func genTestFloatHistogram(timestamp int64, value int) mimirpb.Histogram {
+	return mimirpb.FromFloatHistogramToHistogramProto(timestamp, test.GenerateTestFloatHistogram(value))
+}
+
+func TestIngesterStreamingMixedResultsHistograms(t *testing.T) {
+	// TODO(histograms): Add similar test for mixed samples and histograms?
+	const (
+		mint = 0
+		maxt = 10000
+	)
+	// TODO(histograms): define merge behaviour when float and integer histograms are at the same timestamp? currently it prefers the one in s1
```

Comment:
I think we've concluded that this should not happen (https://raintank-corp.slack.com/archives/C04AT2ANL90/p1675961866885109?thread_ts=1675895790.264209&cid=C04AT2ANL90) and the behavior is expected to be undefined, so we can just test whatever the code does currently.